.github/ISSUE_TEMPLATE/data_request.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Data request
+about: Suggest data to add to this platform
+
+---
+
+**Please describe the data requested**
+A clear and concise description of the data needed
+
+**How would this data be visualized?**
+*Takwimu supports the following visualizations histogram, pie chart, column chart, grouped column chart, Bar chart, grouped bar chart and line charts*
+
+
+**Which countries is this data related to?**
+*Select the countries from Burkina Faso, DRC Congo, Ethiopia, Kenya, Nigeria, Senegal, South Africa, Tanzania, Uganda and Zambia.*
+
+
+**Which topics is this data related to?**
+*Select topics from Development Context and Key Issues, Policy Framework, Sector Policies and Programming
+Public Finance, the Budget and Official Development Assistance. If you data does not fit in any other mentioned topics, feel free to suggest a new topic*
```

Comment:
Let's use the World Bank data topics:

> Agriculture & Rural Development, Aid Effectiveness, Climate Change, Economy & Growth, Education, Energy & Mining, Environment, External Debt, Financial Sector, Gender, Health, Infrastructure, Poverty, Private Sector, Public Sector, Science & Technology, Social Development, Social Protection & Labor, Trade, Urban Development

Source: https://data.worldbank.org/indicator

Reply:
@DavidLemayian does that mean we'll also have to update the topics on the country profiles?